dist/exceptionless.d.ts

```diff
@@ -31,6 +31,10 @@ export interface IEnvironmentInfoCollector {
 export interface IErrorParser {
     parse(context: EventPluginContext, exception: Error): IError;
 }
+export interface IExitController {
```

Comment:
Yes! That sounds great. I don't like `IExitController` very much either, but I didn't want to put the `isAppExiting` global state in the config. I already had `processSync` and `postEventsSync`, but this required lots of refactoring. I removed it, as this would have affected the script for the browser too much.

Now that you also suggest adding `processSync` and `postEventsSync`, I think we should still go for it. Still don't like the naming, because the browser wouldn't need to do it "synchronously": There is some experimental tech called [`sendBeacon`](https://developer.mozilla.org/de/docs/Web/API/Navigator/sendBeacon) (sadly not `sendBacon`, by the way) that would be exactly what we need: Sending stuff even after the page has been unloaded, thereby not blocking navigation. That would be another case where we don't get a 202, it's fire-and-forget.

So, how should we call it?